scalafix-nsc/src/test/scala/cats/data/Xor.scala

```diff
@@ -1,10 +1,17 @@
 package cats.data
+import scala.language.higherKinds
 
-sealed abstract class Xor[+A, +B] extends Product with Serializable
+sealed abstract class Xor[+A, +B] extends Product with Serializable {
```

Comment:
Stub out enough of cats.data.Xor and XorT interface to be able to test rewrites without needing to add a dependency on cats